docs/provision_kind.md

```diff
@@ -0,0 +1,148 @@
+# Provisioning a Kind (Kubernetes in Docker) Cluster
+
+[kind](https://github.com/kubernetes-sigs/kind) is a lightweight tool for running local Kubernetes clusters using containers.  It can be used as the underlying infrastructure for Functions, though it is intended for testing and development rather than production deployment.
+
+This guide walks through the process of configuring a kind cluster to run Functions with the following vserions:
+* kind   v0.8.1 - [Install Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+* Kubectl v1.17.3 - [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) 
```

Comment:
Don't you need to explicitly install Knative serving/eventing?

Reply:
Yes, Kind is a vanilla Kubernetes so it includes no Knative Serving or Eventing.  I currently have the docs segmented out such that provisioning the cluster is separate from configuring it, because regardless of how one arrives at a functional cluster, the steps necesary to install and configure KNative, CertManager, etc. are the same.

However, there ended up being a couple of "gotchas" which make configuring a local cluster (Kind) different from configuring a cloud-based cluster such as EKS.  Therefore, when taken along with your comment here, it solidifies the choice to replicate those configuraiton instructions in each of the cluster types, trading clarity for a bit of redundancy.  I will make this change on the next pass.